lib/linguist/languages.yml

```diff
@@ -348,8 +348,10 @@ Apex:
   type: programming
   color: "#1797c0"
   extensions:
+  - ".apex"
   - ".cls"
-  tm_scope: source.java
+  - ".trigger"
```

Comment:
This is going to cause you problems as this extension is very popular and not unique to Apex. If you want to add this as part of this PR, you will need to identify at least one other language that uses this extension and add support to this PR, along with at least two unique samples to help the classifier.

The the other language isn't sufficiently unique from Apex, then a heuristic will be needed too.

Reply:
I added `.trigger` to the `shell` extension, as that's where it's commonly used.

Reply:
We're going to need at least two samples from each language too.